.ci.yaml

```diff
@@ -2345,6 +2345,18 @@ targets:
       task_name: android_lifecycles_test
     scheduler: luci
 
+  - name: Linux_android openpay_benchmarks__scroll_perf
+    bringup: true
+    recipe: devicelab/devicelab_drone
+    presubmit: false
+    timeout: 60
+    properties:
+      openpay: true
```

Comment:
Looks like this is not the right way to add the properties.  Is there a pointer to the right way? or do we need to fix the CI.yaml validation check?